x-pack/plugins/spaces/common/licensing/license_service.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This is likely overkill for the needs of this PR, but I wanted to set up Spaces to have a similar license service as Security, for consistency. If Spaces ever gets more complex, we'll already be setup to support it.